pynest/nest/__init__.py

```diff
@@ -44,6 +44,8 @@
 except ImportError:
     pass
 
+__version__ = version()
```

Comment:
```suggestion
__version__ = ll_api.sli_func("statusdict /version get")
```
If this works, `version()` could be safely removed, but there might be help texts that point to it and need to be adapted as well.

Reply:
It works! I've removed `version()` and adapted the help text.